webpack/assets/javascripts/react_app/actions/NotificationActions.js

```diff
@@ -1,18 +1,27 @@
 import API from '../API';
 import AppDispatcher from '../dispatcher';
-import {ACTIONS} from '../constants';
+import NotificationsStore from '../stores/NotificationsStore';
+import { ACTIONS, STATUS } from '../constants';
 
 const TIMER = 10000;
 
 export default {
   getNotifications(url) {
-    if (document.visibilityState === 'visible') {
+    if (document.visibilityState !== 'hidden' &&
```

Comment:
@gailsteiger i dont think so - see https://github.com/turbolinks/turbolinks#full-list-of-events